Week4/app/src/main/java/com/em4n0101/mymovies/MovieDetailFragment.kt

```diff
@@ -0,0 +1,60 @@
+package com.em4n0101.mymovies
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.lifecycle.ViewModelProvider
+import com.em4n0101.mymovies.data.Movie
+import com.em4n0101.mymovies.utils.Utilities
+import kotlinx.android.synthetic.main.fragment_movie_detail.*
+
+class MovieDetailFragment : Fragment() {
+
+    private lateinit var moviesManager: MoviesViewModel
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_movie_detail, container, false)
```

Comment:
You can pass layout resource in the constructor of the Fragment so you don't have to override `onCreateView()`

```kotlin
class MovieDetailFragment : Fragment(R.layout.fragment_movie_detail)
```